.github/workflows/github-release.yml

```
@@ -212,8 +212,8 @@ jobs:
       - name: Checkout the merged commit from PR and base branch
         uses: actions/checkout@v4
 
-      - name: Set up Depot CLI
-        uses: depot/setup-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
```

actionlint (1.7.4)

the runner of "docker/setup-buildx-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)